lte/gateway/deploy/agw_install_ubuntu.sh

```diff
@@ -24,7 +24,7 @@ DEPLOY_PATH="/home/$MAGMA_USER/magma/lte/gateway/deploy"
 SUCCESS_MESSAGE="ok"
 NEED_REBOOT=0
 WHOAMI=$(whoami)
-MAGMA_VERSION="${MAGMA_VERSION:-v1.7}"
+MAGMA_VERSION="${MAGMA_VERSION:-v1.8}"
```

Comment:
your PR is called "updated **docker** install script for agw", but it also updates debian install?

Reply:
Reverted this change.